src/global/vars.h

```diff
@@ -296,4 +296,7 @@ extern int16_t InvChosen;
 
 extern int16_t BarOffsetY[6];
 
+extern int32_t LaraSpeedError;
+extern int16_t ANIM_SCALE;
```

Comment:
Global variables should be named in PascalCase. This looks like a macro.

Reply:
it was a macro, but I made it a toggle. I can do a find and replace but it will modify a lot.